prometheus/README.md

```diff
@@ -45,6 +47,8 @@ If `send_monotonic_counter: True`, the Agent sends the deltas of the values in q
 
 All metrics collected by the prometheus check are forwarded to Datadog as custom metrics.
 
+Note: Bucket data for a given `<HISTOGRAM_METRIC_NAME>` Prometheus histogram metric are stored in the `<HISTOGRAM_METRIC_NAME>.count` metric within Datadog with the tags `upper_bound` including the name of the buckets. To access the `+Inf` bucket, use `upper_bound:none`.
```

Comment:
This line seems not used anymore: `[1]: https://docs.datadoghq.com/developers/metrics/custom_metrics`

Reply:
Fixed